cmd/apps/portainer_app.go

```diff
@@ -5,97 +5,103 @@ package apps
 
 import (
 	"fmt"
-	"io"
-	"net/http"
-	"os"
-	"path"
-	"strings"
-
-	"github.com/alexellis/arkade/pkg/config"
-	"github.com/alexellis/arkade/pkg/k8s"
-
 	"github.com/alexellis/arkade/pkg"
+	"github.com/alexellis/arkade/pkg/apps"
+	"github.com/alexellis/arkade/pkg/config"
+	"github.com/alexellis/arkade/pkg/types"
 
 	"github.com/spf13/cobra"
 )
 
 func MakeInstallPortainer() *cobra.Command {
-	var command = &cobra.Command{
+	var portainer = &cobra.Command{
 		Use:          "portainer",
 		Short:        "Install portainer to visualise and manage containers",
 		Long:         `Install portainer to visualise and manage containers, now in beta for Kubernetes.`,
 		Example:      `  arkade install portainer`,
 		SilenceUsage: true,
 	}
 
-	command.RunE = func(command *cobra.Command, args []string) error {
+	portainer.Flags().String("namespace", "default", "Namespace for the app")
+	portainer.Flags().Bool("persistence", false, "Use a 10Gi Persistent Volume to store data")
```

Comment:
I would remove the text "10Gi" since that's not something we set, but I guess you picked it up from the helm chart README?

Reply:
Sure, I did something similar on CockroachDB since I think it is important to know how much disk space it can take.
(Thats maybe cause I only rock a "compute node" right now which has 120Gi total.)